msg/RadarPoint.msg

```diff
@@ -0,0 +1,10 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+float32 range                            # Distance (m) from the sensor to the detected point.
+float32 angle                            # Angle (in radians) in the azimuth plane between the sensor and the detected point.
+                                         #   Negative angles are anticlockwise from the sensor.
+float32 elevation                        # Angle (in radians) in the elevation plane between the sensor and the detected point.
+                                         #   Negative angles are below the sensor. For 2D radar, this will be 0.
+float32 doppler                          # The doppler speeds (m/s) of the point.
+uint8 intensity                          # The intensity of the point (0 - 255)
```

Comment:
This needs units.

Reply:
http://docs.ros.org/melodic/api/sensor_msgs/html/msg/LaserScan.html `LaserScan` does not provide those and this is analog.

Reply:
I haven't done much work with radar drivers, but I would think that `snr` or `amplitude` would make sense (which is what intensity is a surrogate for?)  It could be a parallel field, but I think it should probably be `float32`

Reply:
I agree that "amplitude" would be a more fitting name for this field. The units for an amplitude field would be `dB`.